src/Servers/HttpSys/src/HttpSysOptions.cs

```diff
@@ -68,8 +68,11 @@ public string RequestQueueName
         public ClientCertificateMethod ClientCertificateMethod { get; set; } = ClientCertificateMethod.AllowCertificate;
 
         /// <summary>
-        /// The maximum number of concurrent accepts.
+        /// The maximum number of concurrent accepts dequeuing requests from the Http.sys queue.
```

Comment:
As someone who doesn't know what this property does, the comment doesn't help 😄

Reply:
LOL. It's *very* advanced but I wanted to at least have the defaults there and explain that its about dequeuing...

Reply:
Is it better?